uavcan/equipment/air_data/1027.RawAirData.uavcan

```diff
@@ -0,0 +1,41 @@
+#
+# Raw Air Data.
+#
+
+# Note: unused vars should be assigned NaN
+
+#
+# Heater State
+# 
+uint8 FLAG_HEATER_AVAILABLE      = 0
+uint8 FLAG_HEATER_WORKING        = 1
+uint8 FLAG_HEATER_OVERCURRENT    = 2
+uint8 FLAG_HEATER_OPENCIRCUIT    = 3
+uint8 flags
+
+#
+# Pressure Data
+#
+float32 static_pressure                 # Pascal
+float32 differential_pressure           # Pascal
```

Comment:
I don't think that would be a good idea, the value can range from ~1 to 300Pa (considering max airspeed of ~25m/s, it can go higher with faster planes). we should atleast give way for 0.01 resolution in the whole range, which I doubt will happen with float16.